### ISSUE TITLE
Wrongly raise `id missing` error when translation === id

### ISSUE THREAD

**Issue by esquevin** [ibitcy/eo-locale#44]
Current code forbids having a translation be the id, and erroneously says that the locale is missing the translation.

```if (message === id) {
        this.onError(new Error(`[eo-locale] id missing "${id}"`));
      }```

Even though using text as id is not a best practice, this error message is wrong.

**Comment by pret-a-porter** [comment]
@esquevin Thank you for your issue. I will fix it.

**Comment by pret-a-porter** [comment]
@esquevin fixed in 1.4.12
